app/assets/stylesheets/utilities/typography.scss

```diff
@@ -0,0 +1,3 @@
+.text-light-grey {
+  color: #6f777b;
```

Comment:
I worked out how to reference the gov-uk colours directly, if that's preferable:
`govuk-colour("grey-1")`